src/main/java/net/tiagofar78/prisonescape/managers/ConfigManager.java

```diff
@@ -73,4 +73,20 @@ public Hashtable<PrisonEscapeLocation, PrisonEscapeLocation> getPrisionersSecret
 	public Hashtable<PrisonEscapeLocation, PrisonEscapeLocation> getPoliceSecretPassageLocations() {
 		return null; // TODO
 	}
+
+	public int getCommonItemsProbability() {
+		return 0; // TODO
+	}
+
+	public int getRareItemsProbability() {
+		return 0; // TODO
+	}
+
+	public int getNumberOfChests() {
```

Comment:
This method should not exist. It will be really useless since the chests are on a list.